addons/sale_timesheet/wizard/project_create_sale_order.py

```diff
@@ -50,7 +50,7 @@ def default_get(self, fields):
 
     sale_order_id = fields.Many2one(
         'sale.order', string="Sales Order",
-        domain="['|', '|', ('partner_id', '=', partner_id), ('partner_id', 'child_of', commercial_partner_id), ('partner_id', 'parent_of', partner_id)]")
```

Comment:
commercial_partner_id is 2 lines above this, is this part of the change necessary?